### ISSUE TITLE
Add a support us section

### ISSUE THREAD

**Issue by michaelbeil** [AffiliateWP/theme#135]
We could add a "Support us" area that mentions that users can support us by buying another license, upgrading their license, or telling a friend. 

There could also be a way for them to send us 🍺 or ☕ money .

**Comment by pippinsplugins** [comment]
Are there known examples of where a product company has done this tastefully?

**Comment by michaelbeil** [comment]
Yes. Flywheel has done this quite splendidly: https://www.dropbox.com/s/b9lxiqunbyc419g/beer-cheer.mov?dl=0

**Comment by michaelbeil** [comment]
That is specifically for the support team.

**Comment by pippinsplugins** [comment]
Oh yeah, that is awesome.

**Comment by michaelbeil** [comment]
I think so too.
